marshmallow_dataclass/__init__.py

```diff
@@ -256,6 +260,53 @@ def class_schema(
     Traceback (most recent call last):
       ...
     TypeError: None is not a dataclass and cannot be turned into one.
+
+    >>> @dataclasses.dataclass
+    ... class Anything:
+    ...     class Meta:
+    ...         pass
+    ...     @marshmallow.validates('name')
+    ...     def validates(self, *args, **kwargs):
+    ...         pass
+    ...     @marshmallow.validates_schema
+    ...     def validates_schema(self, *args, **kwargs):
+    ...         pass
+    ...     @marshmallow.pre_dump
+    ...     def pre_dump(self, *args, **kwargs):
+    ...         pass
+    ...     @marshmallow.post_dump
+    ...     def post_dump(self, *args, **kwargs):
+    ...         pass
+    ...     @marshmallow.pre_load
+    ...     def pre_load(self, *args, **kwargs):
+    ...         pass
+    ...     @marshmallow.post_load
+    ...     def post_load(self, *args, **kwargs):
+    ...         pass
```

Comment:
Maybe just one or two marshmallow hooks would have been enough, in order to keep the doctests small ?

Reply:
Done